TeamCode/src/main/java/com/hfrobots/tnt/season1920/DriverControls.java

```diff
@@ -85,6 +85,12 @@ of this software and associated documentation files (the "Software"), to deal
 
     private OpenLoopMecanumKinematics kinematics;
 
+    private FoundationGripMechanism foundationGripMechanism;
+
+    protected DebouncedButton foundationGripButton;
```

Comment:
If not referenced outside of this class directly (and they shouldn't be), these should have a private visibility modifier.